.slack/slack.json

```diff
@@ -1,17 +1,17 @@
 {
   "manifest": {
     "script": {
-      "default": "deno run -q --unstable --import-map=import_map.json --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.7/mod.ts --manifest"
+      "default": "deno run -q --unstable --config=deno.jsonc --allow-read --allow-net https://deno.land/x/deno_slack_builder@0.0.7/mod.ts --manifest"
```

Comment:
in a future release of deno, we should be able to drop this `--config` flag entirely